<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="5" rowspan="4">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Options</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QCheckBox" name="chkFiles">
         <property name="text">
          <string>Files</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="chkDirs">
         <property name="text">
          <string>Directories</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="chkSubDirs">
         <property name="text">
          <string>Sub directories</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="chkHidden">
         <property name="text">
          <string>Hidden</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="chkSystem">
         <property name="text">
          <string>System</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2" colspan="3">
     <widget class="QComboBox" name="cboFile">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="3">
     <widget class="QComboBox" name="cboText">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="cboPath">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QPushButton" name="btnBrowse">
      <property name="text">
       <string>&amp;Browse...</string>
      </property>
      <property name="icon">
       <iconset resource="../res/resource.qrc">
        <normaloff>:/images/choose_folder.png</normaloff>:/images/choose_folder.png</iconset>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="6">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="lblDir">
      <property name="text">
       <string>In directory:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="lblText">
      <property name="text">
       <string>Containing text:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="lblFile">
      <property name="text">
       <string>Find:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="6">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="btnMenu">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../res/resource.qrc">
          <normaloff>:/images/menu.png</normaloff>:/images/menu.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnFind">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Find</string>
        </property>
        <property name="icon">
         <iconset resource="../res/resource.qrc">
          <normaloff>:/images/find.png</normaloff>:/images/find.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>886</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../res/resource.qrc">
     <normaloff>:/images/menu-about.png</normaloff>:/images/menu-about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>cboFile</tabstop>
  <tabstop>cboText</tabstop>
  <tabstop>cboPath</tabstop>
  <tabstop>btnBrowse</tabstop>
  <tabstop>chkFiles</tabstop>
  <tabstop>chkDirs</tabstop>
  <tabstop>chkHidden</tabstop>
  <tabstop>chkSubDirs</tabstop>
  <tabstop>chkSystem</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>btnMenu</tabstop>
  <tabstop>btnFind</tabstop>
 </tabstops>
 <resources>
  <include location="../res/resource.qrc"/>
 </resources>
 <connections/>
</ui>
